crates/bevy_pbr/src/light.rs

```diff
@@ -39,7 +41,9 @@ impl PointLightUniform {
 
         // premultiply color by intensity
         // we don't use the alpha at all, so no reason to multiply only [0..3]
-        let color: [f32; 4] = (light.color * light.intensity).into();
+        let mut color: [f32; 4] = (light.color * light.intensity).into();
+        color[3] = light.radius;
```

Comment:
could you change the comment to mention the alpha channel will now be used for the radius?

Reply:
Since it looks like you're adding a new vec4 for `inverse_range_squared` then I might just put it there instead, since the area light parameters are likely to expand in the future and it'll help keep them together.